src/ModelContextProtocol.AspNetCore/IdleTrackingBackgroundService.cs

```diff
@@ -94,12 +133,28 @@ private async Task DisposeSessionAsync(HttpMcpSession<StreamableHttpServerTransp
         }
     }
 
+    private sealed class SessionTimestampComparer : IComparer<(string SessionId, long Timestamp)>
+    {
+        public static SessionTimestampComparer Instance { get; } = new();
+
+        public int Compare((string SessionId, long Timestamp) x, (string SessionId, long Timestamp) y) =>
```

Comment:
This appearst to mirror the built-in IComparable<T> implementation for tuples (but with the elements flipped). What specific concern made you want to customize this?

Reply:
Good idea! I didn't even consider using the default sort behavior for tuples. I'm already using default tuple-equality to compare user ID claims.